README.md

```diff
@@ -27,23 +27,33 @@ Consult the [Python SDK reference docs](https://kubeflow-pipelines.readthedocs.i
 
 Refer to the [versioning policy](./docs/release/versioning-policy.md) and [feature stages](./docs/release/feature-stages.md) documentation for more information about how we manage versions and feature stages (such as Alpha, Beta, and Stable).
 
+## Development
```

Comment:
I like the idea!

However, https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md is our source of truth to put all guidelines to contrributions. I think it'll be good to add a link to that file in the root README. and include details in the CONTRIBUTING.md doc

Reply:
That makes sense! I will update the README and move the development detail to CONTRIBUTING file.